server/dataset/cxg_dataset.py

```diff
@@ -244,7 +240,7 @@ def __remap_indices(self, coord_range, coord_mask, coord_data):
         if coord_mask is None:
             return coord_range, coord_data
 
-        indices = np.where(coord_mask)[0]
+        indices = coord_mask.nonzero()[0]
```

Comment:
Out of curiosity, is this more efficient than using `np.where`?

Reply:
no.  They are literally identical, but np.where is deprecated in this case.  See the numpy documentation here: https://numpy.org/doc/stable/reference/generated/numpy.where.html

It says:
```
Note

When only condition is provided, this function is a shorthand for np.asarray(condition).[nonzero](https://numpy.org/doc/stable/reference/generated/numpy.nonzero.html#numpy.nonzero)(). Using nonzero directly should be preferred, as it behaves correctly for subclasses. The rest of this documentation covers only the case where all three arguments are provided.
```